modules/images/regenerate-existing-images/load.php

```diff
@@ -59,7 +64,7 @@ function perflab_register_background_job_taxonomy() {
 	 * `wp_set_object_terms` which do not check if there is relationship
 	 * between object and taxonomy.
 	 */
-	register_taxonomy( 'background_job', array(), $args );
+	register_taxonomy( 'background_job', array( 'post' ), $args );
```

Comment:
See my other comment. This feels a bit like a hack? It shouldn't be tied to the `post` post type.